packages/dds/tree/src/feature-libraries/flex-map-tree/mapTreeNode.ts

```diff
@@ -482,9 +482,31 @@ class EagerMapTreeSequenceField<T extends FlexAllowedTypes>
 	extends EagerMapTreeField<T>
 	implements FlexTreeSequenceField<T>
 {
-	public get editor(): SequenceFieldEditBuilder<ExclusiveMapTree[]> {
-		throw unsupportedUsageError("Editing an array");
-	}
+	public editor: SequenceFieldEditBuilder<ExclusiveMapTree[]> = {
```

Comment:
`readonly`? This was previously a getter-only property.

Reply:
Ya, it should also be made `readonly` on `FlexTreeSequenceField`